src/pe_source/intelx_identity.py

```diff
@@ -0,0 +1,289 @@
+"""Collect Intelx credential leak data."""
+# Standard Python Libraries
+import datetime
+import logging
+import sys
+import time
+
+# Third-Party Libraries
+import numpy as np
+import pandas as pd
+import requests
+
+from .data.pe_db.config import get_params
+from .data.pe_db.db_query_source import (
+    connect,
+    get_data_source_uid,
+    get_intelx_breaches,
+    get_orgs,
+    insert_intelx_breaches,
+    insert_intelx_credentials,
+    org_root_domains,
+)
+
+# Calculate Datetimes for collection period
+TODAY = datetime.date.today()
+DAYS_BACK = datetime.timedelta(days=16)
+START_DATE = (TODAY - DAYS_BACK).strftime("%Y-%m-%d %H:%M:%S")
+END_DATE = TODAY.strftime("%Y-%m-%d %H:%M:%S")
+
+
+section = "intelx"
+params = get_params(section)
+api_key = params[0][1]
+
+LOGGER = logging.getLogger(__name__)
+
+
+class IntelX:
+    """Fetch IntelX data."""
+
+    def __init__(self, orgs_list):
+        """Initialize IntelX class."""
+        LOGGER.info("Initialized IntelX")
+        self.orgs_list = orgs_list
+
+    def run_intelx(self):
+        """Run IntelX api calls."""
```

Comment:
Use proper capitalization:
```suggestion
        """Run IntelX API calls."""
```